content/general/concepts/virtual-machines/virtual-machines.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'Virtual Machines'
+Description: 'Virtual machines are virtual versions of physical computers.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Virtual Machines'
+  - 'Cloud Computing'
+  - 'Security'
+  - 'DevOps'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'paths/computer-science'
+---
+
+A virtual machine, or VM, is a virtual version of a physical computer. VMs can perform the same computing functions as physical computers, such as running applications or storing data. However, VMs do not use physical hardware to perform these tasks. Instead, VMs use virtual, software-defined versions of hardware that are installed on a host server. VMs only exist as code.
+
+A single server or physical computer can run multiple VMs at the same time. Each VM has its own operating system and functions as an independent compute resource, even though they may share the same host server. For example, if you had a computer with macOS, you could use a VM to run a separate instance of the Windows operating system on top of your existing macOS computer.
```

Comment:
```suggestion
A single server or physical computer can run multiple VMs at the same time. Each VM has its own operating system and functions as an independent compute resource, even though they may share the same host server. For example, a computer running macOS could use a VM to run a separate instance of the Windows operating system from the same computer.
```

For Docs entries, let's try to avoid a "you" voice or "we" voice whenever possible, in favor of a more declarative, manual tone.